test/component/DeclarationSchemaSpec.scala

```diff
@@ -43,7 +43,7 @@ class DeclarationSchemaSpec extends ComponentTestSpec
     startMockServer()
     stubAuditService()
     authServiceAuthorizesCSP()
-    startMdgWcoDecService()
+    startMdgWcoDecServiceV2()
```

Comment:
We are only testing V2 schema here do we want to test v1 & v3 as well?

Reply:
please note `//TODO: Confirm with QA's that schema tests like these that give confidence around schema changes are tested manually by QA's`
I would leave it as is.